packages/shared/src/BootcAPI.ts

```diff
@@ -36,9 +37,13 @@ export abstract class BootcApi {
   abstract generateUniqueBuildID(name: string): Promise<string>;
   abstract openLink(link: string): Promise<void>;
   abstract isLinux(): Promise<boolean>;
+  abstract isMac(): Promise<boolean>;
   abstract getUidGid(): Promise<string>;
   abstract loadLogsFromFolder(folder: string): Promise<string>;
   abstract getConfigurationValue(config: string, section: string): Promise<unknown>;
+  abstract launchVM(folder: string, architecture: string): Promise<void>;
+  abstract readFromClipboard(): Promise<string>;
+  abstract stopVM(): Promise<void>;
```

Comment:
only one VM can run ?

Reply:
at the moment yes until #876 is implemented, didn't want to make this POC too big.

we also have to make sure if it's still running in the background, etc.